src/libraries/System.Runtime/tests/System/Reflection/DefaultBinderTests.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static class DefaultBinderTests
```

Comment:
Did not found any specific test for DefaultBinder, I guess it is covered some level by different `Invoke()` APIs tests. So created a new Test file